### ISSUE TITLE
Can't buy an item

### ISSUE THREAD

**Issue by FNGEntertainment** [godot-sdk-integrations/godot-google-play-billing#56]
Hello! Thank you very much for sharing the plugin with us, I have an error and I am using version 2.0.0 of the plugin, the problem is that when I press the buy button, I have two errors:

---
Mistake

This version of the app was not configured to be billed through Google Play.

[Accept]
---
and
---
Mistake

This item you requested is not available for purchase

[Accept]
---
What can be?
I have done the same as in the tutorial, I have followed this
https://www.youtube.com/watch?v=qejP2VwvlYE&t=158s
I would really appreciate an answer ^^

**Comment by NianoTT** [comment]
Did you setup the IAP in the Play Console, and is it "active" there?

**Comment by FNGEntertainment** [comment]
Yes! it's active :(

**Comment by FNGEntertainment** [comment]
Oh wait, do I copy the code of the files like build.gradle to my project? maybe is that the issue

**Comment by FNGEntertainment** [comment]
but let's see, I just saw the .gradle settings files and that's what github has, I see mine and they are different, they don't have those lines of code, should I add them? maybe that's the problem.


And yes, I tried to export from the editor and it doesn't work, nor downloading it from Google, I think it must be the files, I don't know.

**Comment by NianoTT** [comment]
Normally you should not have to edit it as it's autogenerated. 

There must be some step that is missing,.. Did you upload a build to Google Play after you created the IAP? If not, you should do it.

**Comment by FNGEntertainment** [comment]
From what I see, it is not generated automatically, because the github has new lines that I do not have, and I do not understand that thing about the iap that you say, you have discord?

**Comment by FNGEntertainment** [comment]
Yes, I have created the iap and I have uploaded it to Google as a video I said, but I don't know if it makes sense because before I have tried two Google plugins and they worked without having to upload to Google Play, for example admob and google play services (login , achievements etc)

**Comment by NianoTT** [comment]
Come to the Godot discord: https://discord.gg/4JBkykG
#mobile-dev channel

**Comment by kyadalu1** [comment]
Try [this](https://www.youtube.com/watch?v=4ik-8S7nFQ8&list=PLQhQEGkwKZUobJyaYz2pQz8KWm-Pkv3rp&index=16)
